src/test/dbwrapper_tests.cpp

```diff
@@ -10,13 +10,11 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
-using namespace boost::assign; // bring 'operator+=()' into scope
-using namespace boost::filesystem;
-         
+
+namespace io = boost::filesystem;
```

Comment:
Might be better to just keep `using namespace boost::filesystem` or specify the full namespace instead.